pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md

```diff
@@ -0,0 +1,227 @@
+>
+> Si vous utilisez le traitement d'images supportant les applications pour les sauvegardes de base de données MS SQL ou Oracle, les options compatibles avec les applications et restaurations d'éléments ne seront pas prises en charge. Une restauration complète de la machine virtuelle doit être réalisée, cela implique une fenêtre de temps d'arrêt pour tous les utilisateurs de la base de données. Il est impossible de relancer manuellement un échec de sauvegarde immuable. Vous devez exécuter une sauvegarde complète active ou attendre l'exécution de la prochaine sauvegarde planifiée (consultez la [documentation Veeam pour en savoir plus](https://helpcenter.veeam.com/docs/backup/vsphere/vcloud_manage_backup.html?ver=120)).
+
+
+### Étape 1 - Sauvegarde
+
+#### Accéder à la console d'administration VEEAM Backup Data Protection
+
+Le service **Protection des données avec VEEAM** dispose d’un plugin pour sauvegarder des VMs et des vApp à partir de n’importe quel Virtual Data Center (VDC) de l’organisation. Il est disponible au niveau de l'organisation pour tout utilisateur VMware Cloud Director ayant le rôle d'administrateur de l'organisation.
+
+Lorsque vous utilisez l'intégration VCD Protection des données avec VEEAM pour créer des tâches de sauvegarde, vous pouvez choisir n'importe quelle instance de machine virtuelle à partir de n'importe quel datacenter virtuel de l'organisation.
+
+Pour accéder au Portail Protection des données avec VEEAM depuis Cloud Director, connectez-vous au portail client VCD avec un compte vCloud Director disposant des droits appropriés.
+
+Dans le menu de la barre centrale supérieure, cliquez sur `Plus`{.action} et sélectionnez `Protection des données avec Veeam`{.action}
+
+![VCD access to Veeam Backup](images/vcd_veeam_backup.png){.thumbnail}
+
+La fenêtre du Plugin VEEAM VCD s'ouvrira avec un bandeau gris/noir.
+
+![VCD access to Veeam Backup](images/vcd_veeam_backup_repo_2.png){.thumbnail}
+
+### Sauvegarder avec le Pluging Veeam VCD
+
+#### Données incluses dans les sauvegardes
+
+Lorsque Veeam Backup & Replication réalise des sauvegardes de vApp et de VM, il capture en plus les métadonnées de vApp.
+
+Les métadonnées des applications virtuelles (vApp) et VM incluent :
+
+- Les informations générales sur les **vApp** (applications virtuelles), où résident les VMs, telles que : **Nom des vApp**, **descriptions**, **description des VMs**.
+- Les informations sur les réseaux **vApp** et les réseaux d'organisation auxquels le vApp est connecté.
+- Les options de démarrage des **VM** (VM Startup options).
+- Les informations utilisateurs.
+- Le bail (Lease).
+- Le quota.
+- Les Modèles de stockages (Templates).
+
+Les métadonnées vApp/VM sont stockées avec le contenu de la machine virtuelle. La capture des métadonnées vApp/VM est extrêmement importante pour la restauration : sans elle, vous ne serez pas en mesure de restaurer les vApp et les VM vers VMware Cloud Director.
+
+<!--- proofread stops below this point --->
```

Comment:
This part (from line 84 to line 97 can't be proofread by our team.
The information is confusing (the different options/repository/storage offers lack descriptions) and the tone (especially the use of "je") is not appropriate.